doc/radosgw/cloud-sync-module.rst

```diff
@@ -94,7 +94,7 @@ Non Trivial Configuration:
 
 * ``connection`` (container)
 
-Represents a connection to the remote cloud service. Contains ``conection_id`, ``access_key``,
+Represents a connection to the remote cloud service. Contains ``connection_id`, ``access_key``,
```

Comment:
Shouldn't this be ``connection_id`` ? (I mean: shouldn't this have two backticks after "connection_id"?)